### PR TITLE
Update ISv4 with latest instantsearch.js

### PR DESCRIPTION
**Pull request by bsuravech**
ISv4 changes:
- update version with suffix `-v3`
- removed angular from package.json
- removed build-back command as we moved adminBundle to it's own branch

### CODE REVIEW

**Comment by bsuravech** [comment]
@Haroenv I saw that there was a release for instantsearch today for `4.15.0`. Any breaking changes for this release? I've updated the bundle with this new one but I'd like to be sure from your end. Thanks!

**Comment by Haroenv** [comment]
It includes smartSort, non-major versions never have breaking changes @bsuravech :)

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "algoliasearch-extensions-bundle",
-  "version": "4.3.9",
+  "version": "4.4.0-v3",
```
Comment (Haroenv) [inline package.json:3]: this isn't valid semver, what do you want v3 to mean? Valid version would be `4.4.0-v3.0`

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   "description": "Bundle autocomplete.js and instantsearch.js for use in algoliasearch extensions integrations",
   "main": "index.js",
   "scripts": {
     "watch": "webpack --watch -d & http-server dist/ & wait",
     "build-front": "VERSION=`cat package.json | json version` ./scripts/build.sh",
     "build-back": "VERSION=`cat package.json | json version` ./scripts/build-admin.sh",
```
Comment (Haroenv) [inline package.json]: if you're no longer using build-back, maybe remove everything related to it too?

File: readme.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ Bundle [jQuery](http://jquery.com/), [autocomplete.js](https://github.com/algoli
 
 Mostly for internal Algolia usage, publicly available for transparency and ease of deployment.
 
+## Build Branches
+
+* `master` - Magento 1 (front, back)
+* `ISv2` - Magento 2 v1.x (front, back)
+* `ISv4` - Magento 2 v2.x/3.x (front)
+* `adminBundle` - Magento 2 v2.x/3.x (back)
```
Comment (Haroenv) [inline readme.md:12]: make sure to also put this info on the other branches

File: package.json (hunk 1 above)
Comment (bsuravech) [reply package.json:3]: This is just a suffix for extension identification. Not meant for versioning.

File: readme.md (hunk 3 above)
Comment (bsuravech) [reply readme.md:12]: Already added to master. Would be cool to have a Wiki opened for this repo instead so we can manage it that way.

Comment (Haroenv) [reply readme.md:12]: I don't have access to add a wiki, you'll need to ask helpit :)

**Review by Haroenv — APPROVED** [review]

**Pull request closed without merging**